docker-compose.dev.yml

```
@@ -57,7 +57,7 @@ services:
       SOKETI_DEFAULT_APP_ID: "${PUSHER_APP_ID:-coolify}"
       SOKETI_DEFAULT_APP_KEY: "${PUSHER_APP_KEY:-coolify}"
       SOKETI_DEFAULT_APP_SECRET: "${PUSHER_APP_SECRET:-coolify}"
-    entrypoint: ["/bin/sh", "/soketi-entrypoint.sh"]
+    entrypoint: [ "/bin/sh", "/soketi-entrypoint.sh" ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)